ed/idlpatches/css-typed-om.idl.patch

```diff
@@ -0,0 +1,35 @@
+From 19fbd9a43ddae066c864e4dbcd3493cd96e7aadb Mon Sep 17 00:00:00 2001
+From: Francois Daoust <fd@tidoust.net>
+Date: Mon, 17 May 2021 14:21:51 +0200
+Subject: [PATCH] Restore previous definition for CSSColor
+
+New definition references not yet defined `CSSPercentishArray` structure:
+https://github.com/w3c/css-houdini-drafts/commit/8d1990684d86ad7cf1e58ba414114c8811c04ee3
+
+Property should eventually get fixed by:
+https://github.com/w3c/css-houdini-drafts/issues/1037
```

Comment:
maybe, but I think it would be best if there was a separate issue to track the lack of definition for `CSSPercentishArray`

Reply:
I didn't do it because the first commit message alludes to the need to define the type, and the second issue discusses changing the type further, so the group seems already well aware that the interface needs to be updated.

Also, as opposed to the other patch, any update to the interface will invalidate this patch, making is easy to detect when the patch needs to be dropped.

Reply:
issue filed at https://github.com/w3c/css-houdini-drafts/issues/1041